compiler/il/OMRILOps.hpp

```diff
@@ -917,15 +910,12 @@ class ILOpCode
       switch (longOp)
          {
          case TR::ladd:   return TR::iadd;
-         case TR::luadd:  return TR::iuadd;
```

Comment:
again until the opcode is gone luadd ->iadd here IMO

Reply:
Fixed in 1e7b727